packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-authentication.jsx

```diff
@@ -174,7 +174,8 @@ const TwoFactorAuthentication = ({ email_address, is_switching }) => {
                         <Timeline.Item
                             item_title={localize('Enter the authentication code generated by your 2FA app:')}
                         >
-                            <DigitForm is_enabled={is_two_factor_enabled} setEnabled={setEnabled} />
+                            <DigitForm is_enabled={is_two_factor_enabled} setEnabled={setEnabled}
+                             logoutClient={logoutClient}/>
```

Comment:
use prettier extension to format the file.
![image](https://user-images.githubusercontent.com/82078941/168016732-13b8fc52-3021-452e-822b-2ca1ba34c512.png)